api/capi/include/nntrainer.h

```diff
@@ -180,8 +180,7 @@ int ml_nnlayer_delete(ml_nnlayer_h layer);
  * @retval #ML_ERROR_NONE Successful.
  * @retval #ML_ERROR_INVALID_PARAMETER Invalid parameter.
  */
-int ml_nnlayer_set_property(ml_nnlayer_h layer, const char* key,
-                            const char *value);
+int ml_nnlayer_set_property(ml_nnlayer_h layer, const char *key, ...);
```

Comment:
Since param `value` is removed, `@param[in]  value Property value` comment is not valid.

Reply:
yes. Thanks.!